plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hadoop/HadoopIcebergTableOperations.java

```diff
@@ -61,13 +69,34 @@ protected String getRefreshedLocation(boolean invalidateCaches)
     @Override
     protected void commitNewTable(TableMetadata metadata)
     {
-        // TODO Auto-generated method stub
+        String newMetadataLocation = writeNewMetadata(metadata, version + 1);
```

Comment:
This is probably my lack of Iceberg knowledge, but why the `+1`?

Reply:
I'm not entirely sure to be honest, but it's what the other connectors did and it works so I'm going to assume it's the right thing to do.